Makefile.am

```diff
@@ -26,25 +26,35 @@ data_DATA=etc/openfortivpn/config.template
 
 etc/openfortivpn/config: $(srcdir)/etc/openfortivpn/config.template
 	@$(MKDIR_P) etc/openfortivpn
-	$(AM_V_GEN)$(SED) -e '3,$$ s/^/# /' $(srcdir)/etc/openfortivpn/config.template >$@
+	$(AM_V_GEN)$(SED) -e '3,$$ s|^|# |' $(srcdir)/etc/openfortivpn/config.template >$@
 
-install-data-hook: etc/openfortivpn/config
+lib/systemd/system/openfortivpn@.service: $(srcdir)/lib/systemd/system/openfortivpn@.service.in
+	@$(MKDIR_P) lib/systemd/system
+	$(AM_V_GEN)$(SED) -e 's|[@]SYSCONFDIR[@]|$(sysconfdir)|g' $(srcdir)/lib/systemd/system/openfortivpn@.service.in >$@
+
+install-data-hook: etc/openfortivpn/config lib/systemd/system/openfortivpn@.service
 	if ! test -f $(DESTDIR)$(confdir)/config ; then \
 		$(MKDIR_P) $(DESTDIR)$(confdir) ; \
 		$(INSTALL) -m 600 etc/openfortivpn/config \
 			$(DESTDIR)$(confdir)/config ; \
 	fi
+if HAVE_SYSTEMD
```

Comment:
I wasn't aware that such an if construct works in the .am file, but if it works, why not...

Reply:
Yes, I've seen it used in other Makefiles, not certain how standard this is. If you have alternatives I'm interested.

Reply:
I would have used `AC_SUBST` in configure.ac and compare it as a string with `test` inside `Makefile.am` - but I'm fine with the `if HAVE_SYSTEMD`. It's more elegant, I just didn't know about this possibility.